### DISCUSSION TITLE
Welcome to templUI Discussions!

### DISCUSSION THREAD

**Discussion by axadrn** [templui/templui#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dimmerz92** [comment]
Hey @axzilla ,

Thanks for publishing this library, I haven't used it yet but I am looking to incorporate it into a small project I am building now.

Being more of a back-end / data person, I am creatively and artistically challenged and find it quite difficult to design front-ends on my own. I've learnt over time that a good UI/UX person is truly worth their weight in gold! Libraries like this allow me to quickly get things developed while providing a nice functional user experience that can be better refined over time.

Having said all that, I'd love to contribute in any way I can. I would love to see this project become successful in the Go web dev community as it provides huge value in my opinion. Feel free to reach out.

Cheers,
Andrew

**Reply by axadrn** [reply]
Hey @dimmerz92,

Thanks for these super kind words. This makes me really happy. I totally understand the UI/UX challenges coming from a backend background - it's exactly why I created this library, to help developers build great features while having a solid foundation for the frontend.

Really appreciate your offer to contribute! The project is open to all kinds of contributions, from bug reports to code. Feel free to open issues or PRs on GitHub if you have ideas while using it in your project. Don't hesitate to reach out if you need any help!

Cheers,
Axel

**Reply by dimmerz92** [reply]
No worries!, I might make some PR's in the future after I've mastered the centering of a div.
